src/app/shared/issue/duplicateOf/duplicate-of.component.ts

```diff
@@ -24,11 +34,33 @@ export class DuplicateOfComponent implements OnInit {
   @ViewChild(MatSelect) duplicateOfSelection: MatSelect;
   @ViewChild(MatCheckbox) duplicatedCheckbox: MatCheckbox;
 
+  // Max chars visible for a duplicate entry in duplicates dropdown list.
+  readonly MAX_TITLE_LENGTH_FOR_DUPLICATE_ISSUE = 17;
+  // Max chars visible for a non-duplicate entry in duplicates dropdown list.
+  readonly MAX_TITLE_LENGTH_FOR_NON_DUPLICATE_ISSUE = 37;
+
   constructor(public issueService: IssueService,
               private errorHandlingService: ErrorHandlingService,
               public permissions: PermissionService) {
   }
 
+  /**
+   * Checks if the supplied issue requires a tooltip
+   * in the UI as some information may be hidden due to truncation.
+   * @param issue - Displayed issue that may need a tooltip.
+   * @return - true (to enable tooltip) / false (to disable tooltip)
+   */
+  isTooltipNecessary(issue: Issue): boolean {
+    // Maximum Possible Title length varies based on whether the issue
+    // is a duplicate. (Whether the Duplicate Issue Tag is visible)
+    let maxTitleLength: number;
+    issue.duplicated ?
+      maxTitleLength = this.MAX_TITLE_LENGTH_FOR_DUPLICATE_ISSUE
+      : maxTitleLength = this.MAX_TITLE_LENGTH_FOR_NON_DUPLICATE_ISSUE;
```

Comment:
Does that work too? 
```js
issue.duplicated
  ? maxTitleLength = this.MAX_TITLE_LENGTH_FOR_DUPLICATE_ISSUE
  : maxTitleLength = this.MAX_TITLE_LENGTH_FOR_NON_DUPLICATE_ISSUE;
```

Or this?
```js
maxTitleLength = issue.duplicated
  ?  this.MAX_TITLE_LENGTH_FOR_DUPLICATE_ISSUE
  :  this.MAX_TITLE_LENGTH_FOR_NON_DUPLICATE_ISSUE;
```

Reply:
Yup the second one works too. @ptvrajsk use the more concise version

Reply:
Sure